vuepress/docs/next/docs/reference/entando-apis.md

```diff
@@ -5,28 +5,53 @@ sidebarDepth: 2
 
 ## Overview
 
-Entando includes the Swagger UI in a Getting Started environment and is reachable at `/entando-de-app/api/swagger-ui.html`:
+Entando includes the Swagger UI in a quickstart environment and is reachable at `/entando-de-app/api/swagger-ui.html`:
 
     http://[your-host-name]/entando-de-app/api/swagger-ui.html
 
+### Enable or disable the Swagger UI in a running container
+
+The Swagger UI can be enabled or disabled in a running container by modifying the SPRING_PROFILES_ACTIVE environment variable for the entando-de-app container. 
+
+1. Edit the deployment. The name may be different outside of a quickstart environment.
+```
+sudo kubectl -n entando edit deployment/quickstart-server-deployment
+```
+
+2. Scale the deployment `spec.replicas` to 0 before updating the deployment. This helps avoid database errors that can happen on an immediate restart after the profile is changed. Save the edit to apply the change. 
```

Comment:
This is only true/required in an environment with in memory databases. Might be worth clarifying.

Reply:
Good to know. Updated.